docs-ref-services/data-lake-store.md

```diff
@@ -18,17 +18,37 @@ ms.service: Data Lake Store
 ## Overview
 Azure Data Lake Store is an enterprise-wide hyper-scale repository for big data analytic workloads. Azure Data Lake enables you to capture data of any size, type, and ingestion speed in one single place for operational and exploratory analytics.
 
-## Install the modules with npm
+## Management Package
+
+### Install the npm module
 
 Use npm to install the Azure Data Lake Store modules for Node.js
 
-### Management
 ```bash
 npm install azure-arm-datalake-store
 ```
 
-##Example
+### Example
+
+This example lists all Data Lake Store accounts within a given Azure subscription
+
+```js
+const msRestAzure = require('ms-rest-azure');
+const adlsManagement = require('azure-arm-datalake-store');
+
+const subscriptionId = 'your-subscription-id';
+
+msRestAzure.interactiveLogin().then(credentials => {
+  const accountClient = new adlsManagement.DataLakeStoreAccountClient(
+    credentials,
+    subscriptionId
+  );
+  accountClient.account.list().then(result => {

```

Comment:
I've been returning promises from `then`s and chaining another `then` to handle it (rather than nesting), e.g. https://github.com/jawache/azure-docs-sdk-node/blob/bd56729e353881cb979d8d2614b62c643ef3d78d/docs-ref-services/commerce.md